Package.swift

```diff
@@ -572,7 +572,7 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         // The 'swift-argument-parser' version declared here must match that
         // used by 'swift-driver' and 'sourcekit-lsp'. Please coordinate
         // dependency version changes here with those projects.
-        .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: "1.0.1")),
+        .package(url: "https://github.com/apple/swift-argument-parser.git", branch: "nate/propagate_unconditional_remaining"),
```

Comment:
This is obviously temporary.  Now I'm going to see what the failures are without this fix, and then see whether this PR should land without it.